### PR TITLE
Update CONTRIBUTING and AUTHORING_GUIDE with submission and review guidelines

### PR DESCRIPTION
**Pull request by busunkim96**

### CODE REVIEW

File: AUTHORING_GUIDE.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ owner to review your code for product-specific needs.
 In **library repositories** with pre-existing samples GitHub will automatically assign a reviewer
 from python-samples-owners. If no reviewer is automatically assigned, contact [@googleapis/python-samples-owners](https://github.com/orgs/googleapis/teams/python-samples-owners).
 
-Feel free to reach out to your assigned reviewer if it's been a few days and you haven't gotten a response!
+Please reach out to your assigned reviewer if it's been more than 2 days and you haven't gotten a response!
```
Comment (busunkim96) [inline AUTHORING_GUIDE.md:49]: Switched to "2 days" to match [this doc](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/reviews.md?cl=head#samples-reviewers).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+2. A repo owner will respond to your issue promptly. If you don't see a response within
+   a few days, please ping the owner assignd to your issue.
+3. If your proposed change is accepted, and you haven't already done so, sign a
+   Contributor License Agreement (see details above).
+5. Fork this repo, develop and test your code changes. Tests are required for all
+   samples. See the [Authoring Guide](AUTHORING_GUIDE.md) for details.
+6. Ensure that your code adheres to the existing style in the sample to which
+   you are contributing.
+7. Ensure that your code has an appropriate set of unit tests which all pass.
+8. Submit a pull request.
```
Comment (leahecole) [inline CONTRIBUTING.md:20]:
```suggestion
4. Fork this repo, develop and test your code changes. Tests are required for all
   samples. See the [Authoring Guide](AUTHORING_GUIDE.md) for details.
5. Ensure that your code adheres to the existing style in the sample to which
   you are contributing.
6. Ensure that your code has an appropriate set of unit tests which all pass.
7. Submit a pull request.
```

Comment (leahecole) [reply CONTRIBUTING.md:20]: Total non blocking nit - I know markdown renders this correctly

**Review by leahecole — APPROVED** [review]

**Review by engelke — APPROVED** [review]
LGTM

**Review by dandhlee — APPROVED** [review]
Yay!

**Pull request merged**